lib/config/config.go

```diff
@@ -75,6 +77,33 @@ func GetRemoteOriginUrl() string {
 	return util.GetCommandOutput("git", "remote", "get-url", "origin")
 }
 
+func GetRemoteOriginHostname() string {
+	url := GetRemoteOriginUrl()
+	hostnameRegex, err := regexp.Compile("(^[^:]*://([^@]*@)?|git@)([^/:]+).*")
+	if err != nil {
+		log.Fatal("Error compiling hostname regular expression: ", err)
+	}
+	matches := hostnameRegex.FindStringSubmatch(url)
+	if matches == nil {
+		return ""
+	}
+	return matches[3]
+}
+
+func GetRemoteRepositoryName() string {
+	url := GetRemoteOriginUrl()
+	hostname := GetRemoteOriginHostname()
```

Comment:
This is calling `GetRemoteOriginUrl` twice  in a row. This happens in a number of places in this code base. The switch to the simpler helper methods that I have proposed fixes this as well.